persistence/src/commonMain/kotlin/com/wire/kalium/persistence/backup/BackupImporter.kt

```diff
@@ -0,0 +1,53 @@
+package com.wire.kalium.persistence.backup
+
+import app.cash.sqldelight.db.SqlDriver
+
+interface BackupImporter {
+    suspend fun importFromFile(filePath: String)
+}
+
+class BackupImporterImpl(private val sqlDriver: SqlDriver) : BackupImporter {
+
+    // TODO: Emit steps to display progress as backup is imported
+    override suspend fun importFromFile(filePath: String) {
+        sqlDriver.execute(null, """ATTACH ? AS $BACKUP_DB_ALIAS""", 1) {
+            bindString(1, filePath)
+        }
+        sqlDriver.execute("""BEGIN""")
+
+        migrateTable("Team")
+        migrateTable("User")
+//         migrateTable("SelfUser")
```

Comment:
Why aren't we migrating the `SelfUser` table?

Reply:
I'll add a TODO as for now